### DISCUSSION TITLE
Debug Stored Procedure in DBeaver vers. 22.2.4

### DISCUSSION THREAD

**Discussion by bpapa1965** [dbeaver/dbeaver#23133] [DBeaver Development]
Hi, please in my DBeaver, has a connection DB with Firebird and I'have any stored procedure, 
How do I debug these? Do I need to install something? or are there DBeaver modules that give me this possibility? Can you guide me step by step? Thank you

**Comment by LonwoLonwo** [comment]
Hello @bpapa1965 

DBeaver supports debugger only for PostgreSQL database: https://github.com/dbeaver/dbeaver/wiki/PGDebugger

**Comment by bpapa1965** [comment]
Thanks but if I'want to debugger Stored Procedure with DB MariaDB  a ? It'is possible ?  What tools should I use?
thanks
bye
